spaces/S000088/properties/P000022.md

```diff
@@ -3,9 +3,12 @@ space: S000088
 property: P000022
 value: true
 refs:
-- doi: 10.1007/978-1-4612-6290-9_6
+- zb: "0386.54001"
   name: Counterexamples in Topology
 ---
 
-Asserted in the General Reference Chart for space #90 in
-{{doi:10.1007/978-1-4612-6290-9_6}}.
+Note that {S79|P22}.
+Therefore given a continuous function
+$f:X\to\mathbb R$, for each $n\in\mathbb Z$ one can define
+$c_n=\sup\{|f(x)|: x\in q(Y\times\{n\})\}$, where $q:Z\to X$ is the quotient map.
+But since $\lim_{n\to\pm\infty} c_n= f(a^\pm)$ we obtain the global bound for $f$.
```

Comment:
I think it should be the absolute value of $f(a^\pm)$.
Also, probably  no need for reference to S&S